app/views/home/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <div class="navbar-end">
           <% if user_signed_in? %>
 
-            <a href="#" class="navbar-item">
+            <a href="/inventory" class="navbar-item">
```

Comment:
I'd suggest using `link_to` rails helper for anchors!
Take a look at [this](https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to)

Reply:
Thanks, Luis. That link is great! Should I address this issue in this branch and PR o should I create another branch/PR?

Reply:
I'd suggest changing it now, since you've already modified this lines 👀

Reply:
That 👆🏻
Do no't add technical dept if you can prevent it instead